README.md

```diff
@@ -367,6 +367,30 @@ Example:
 }
 ```
 
+### Special GDD Type: "Invoke Action"
+
+Some templates support invoking custom functions during their life-time.
+This GDD Type should be rendered as a Button in the GUI so that the user can click it to invoke the action.
+
+
+Example:
+```typescript
+{
+  "title": string, // [Mandatory] A short title / label of the action
+  "description": "", // [Optional] A longer description of the action
+  "type": "null",
+  "gddType": "invoke-action",
+  "gddOptions": {
+    // [mandatory] The function to invoke in the template
+    // Example: myCustomHello("world", $path)
+    "invoke": string
+  }
+}
+```
+
+Note: `$path` is a special token which will be substituted at runtime with the path of the action. This is useful when having an action inside an array, since this would cause multiple actions to render (one per row in the data). An example of a path would be `people.2` for the action in the 3rd row of an array called `people`.
```

Comment:
Since JSON Schema has it's own meaning for ["Path"](https://json-schema.org/understanding-json-schema/structuring.html#json-pointer), I think it would be good to use a consistent specification of how "path" is built. I would suggest specifying that this is a JSON Path, as specified by [Stefan Goessner's JsonPath](http://goessner.net/articles/JsonPath/), using the dot notation, without using the root symbol.

Reply:
Very good point!